### ISSUE TITLE
Add file api (File, FileReader, FileWriter examples) slide

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#223]
```
Add file api (File, FileReader, FileWriter examples) slide
```

Original issue reported on code.google.com by `erne...@chromium.org` on 6 Oct 2010 at 7:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 19 Nov 2010 at 3:38
- Added labels: **NewArticle**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 49b11e2ee0.
```

Original comment by `ericbide...@html5rocks.com` on 9 Feb 2011 at 12:04
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 9 Feb 2011 at 12:13
- Added labels: **Type-Feature**, **Priority-P2**, **Milestone-5.2**, **new**, **MovedFrom4**
- Removed labels: **Type-Enhancement**, **Priority-Medium**, **Milestone-4**, **NewArticle**
